src/main/java/org/opentripplanner/middleware/triptracker/ManageTripTracking.java

```diff
@@ -203,12 +226,14 @@ public static TrackingResponse rerouteTracking(Request request) {
     /**
      * Attempt to reroute from the traveler's current location to the trip's original destination.
      */
-    public static boolean rerouteTrip(TripTrackingData tripData) {
+    public static Itinerary rerouteTrip(TripTrackingData tripData) {
         if (tripData != null) {
             var trackedJourney = tripData.journey;
-            return trackedJourney != null && updateTripWithNewItinerary(trackedJourney);
+            if (trackedJourney != null) {
+                return updateTripWithNewItinerary(trackedJourney);
```

Comment:
We should probably notify companions/observers of that change.

Reply:
Addressed inhttps://github.com/ibi-group/otp-middleware/pull/286/commits/23d04728e3e4d229384d0e37b6ca821b9f416208/ @binh-dam-ibigroup can you provide the French please?!